# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qt6webengine/hotfix-glibc-2.43.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- qtwebengine-everywhere-src-6.10.2/src/3rdparty/chromium/sandbox/linux/system_headers/linux_seccomp.h.vanilla	2026-02-07 21:12:09.428272606 +0100
+++ qtwebengine-everywhere-src-6.10.2/src/3rdparty/chromium/sandbox/linux/system_headers/linux_seccomp.h	2026-02-07 21:12:20.191816046 +0100
@@ -215,7 +215,7 @@
 #endif
 
 #ifndef SYS_SECCOMP
-#define SYS_SECCOMP                   1
+//#define SYS_SECCOMP                   1
 #endif
 
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_SECCOMP_H_
